be/src/vec/functions/array/function_array_index.h

```diff
@@ -217,7 +217,7 @@ class FunctionArrayIndex : public IFunction {
         const auto& right_chars = reinterpret_cast<const ColumnString&>(right_column).get_chars();
 
         // prepare return data
-        auto dst = ColumnVector<ResultType>::create(offsets.size());
+        auto dst = ColumnVector<ResultType>::create(offsets.size(), 0);
         auto& dst_data = dst->get_data();
         auto dst_null_column = ColumnUInt8::create(offsets.size());
```

Comment:
need to set default value to 0 like other function.

Reply:
done!